lib/Agrammon/Model.pm6

```diff
@@ -516,4 +535,45 @@ class Agrammon::Model {
         };
         %$!distribution-map-cache
     }
+
+    #| Get a CSV formatted template of all model inputs
+    #| with one demo instance for all multi-instance modules.
+    method get-input-template($sort) {
+        # Obtain model structure in specified order.
+        my @model-structure := self.extract-structure-sorted($sort);
```

Comment:
Here we could instead do `my @modules := $sort eq 'model' ?? @!load-order !! @!evaluation-order`, and...